llama-index-integrations/llms/llama-index-llms-databricks/llama_index/llms/databricks/base.py

```diff
@@ -45,3 +45,11 @@ def __init__(
     def class_name(cls) -> str:
         """Get class name."""
         return "Databricks"
+
+    def _get_model_kwargs(self, **kwargs: Any) -> Dict[str, Any]:
+        """Get the kwargs that need to be provided to the model invocation."""
+        # Fix the input to work with the Databricks API
+        if "tool_choice" in kwargs and "tools" not in kwargs:
+            del kwargs["tool_choice"]
```

Comment:
Tool choice tells the API that the LLM MUST used that tool, at least with regular openai